services/server/src/routes/report.ts

```diff
@@ -0,0 +1,10 @@
+import report from "@/handlers/report";
+import { Router } from "express";
+
+const router = Router();
+
+router.get("/list", report.find);
+router.post("/", report.create);
```

Comment:
Let's follow the same pattern used [here](https://github.com/litespace-org/litespace/blob/master/services/server/src/routes/user.ts#L21-L25).